src/assets/translations.data.json

```diff
@@ -310,6 +313,8 @@
     "client-info.mute-button": "Mute",
     "client-info.cancel": "Cancel",
     "avatar-landing.select": "Select",
-    "avatar-landing.selected": "This is your current avatar"
+    "avatar-landing.selected": "This is your current avatar",
+    "embed.load-button": "Load Room",
+    "embed.presence-warning": "This room is embedded, so may be visible to visitors on a non-Mozilla site."
```

Comment:
maybe "another site?" I feel more like we can claim that it's visible on another site, but not necessarily say much about the nature who can see that site. For example I could imagine if we say that it could be visible by the general public we could scare people who are actually using it on a private, locked down page. (Reminiscent of the `SOMEONE COULD BE DOING SOMETHING NASTY` warning on ssh :))

Reply:
Ah, good point. "visitors on other websites" sounds good.